lib/system/comparisons.nim

```diff
@@ -253,7 +253,9 @@ proc clamp*[T](x, a, b: T): T =
   ## Limits the value `x` within the interval [a, b].
   ## This proc is equivalent to but fatser than `max(a, min(b, x))`.
   ## 
-  ## **Note:** `a <= b` is assumed and will not be checked.
+  ## ..warning:: `a <= b` is assumed and will not be checked.
+  ## **See also:**
+  ## `math.clamp` for a version that takes a Slice[T] instead.
```

Comment:
```suggestion
  ## `math.clamp` for a version that takes a `Slice[T]` instead.
```

(in future work, the compiler might be able to transform things like `Slice[T]` into a hyperlink in some cases)